src/main/java/seedu/address/ui/ModuleCard2.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import seedu.address.model.module.Module;
+
+/**
+ * An UI component that displays information of a {@code Module}.
+ */
+public class ModuleCard2 extends UiPart<Region> {
```

Comment:
Also I'm wondering if these panels should be called `GradedModules` and `UngradedModules` instead? If so, can just change in next PR.